src/main/java/ch/njol/skript/classes/data/BukkitClasses.java

```diff
@@ -1836,5 +1837,34 @@ public String toVariableNameString(Attribute a) {
 					}
 				})
 				.serializer(new EnumSerializer<>(Attribute.class)));
+
+		if (Skript.classExists("io.papermc.paper.world.MoonPhase")) {
+			EnumUtils<MoonPhase> moonPhases = new EnumUtils<>(MoonPhase.class, "moon phases");
+			Classes.registerClass(new ClassInfo<>(MoonPhase.class, "moonphase")
+				.user("(lunar|moon) ?phases?")
+				.name("Moon Phase")
+				.description("Represents the phase of a moon.")
+				.usage(moonPhases.getAllNames())
+				.since("INSERT VERSION")
+				.requiredPlugins("Paper 1.16+")
+				.parser(new Parser<MoonPhase>() {
+					@Override
+					@Nullable
+					public MoonPhase parse(String input, ParseContext context) {
+						return moonPhases.parse(input);
+					}
+
+					@Override
+					public String toString(MoonPhase moonPhase, int flags) {
+						return moonPhases.toString(moonPhase, flags);
+					}
+
+					@Override
+					public String toVariableNameString(MoonPhase moonPhase) {
+						return toString(moonPhase, 0);
```

Comment:
```suggestion
						return moonPhase.name();
```
I think this is preferred? would be consistent with your enum class info pr too